composer.json

```diff
@@ -25,7 +25,7 @@
         }
     },
     "require": {
-        "folklore/graphql": "0.*",
```

Comment:
I changed this in case I needed to make changes to the base package. I have a fork of that one too. Turns out all of this could be done without modifying that package at all. Before hitting another release I'll be sure to change this back.